azure-pipelines.yml

```diff
@@ -296,27 +296,27 @@ stages:
     steps:
       - template: eng/pipelines/checkout-windows-task.yml
 
-      - script: eng/test-build-correctness.cmd -configuration Release -enableDumps
-        displayName: Build - Validate correctness
+      - template: eng/pipelines/build-bootstrap.yml
+        parameters:
+          bootstrapToolset: 
 
-      - template: eng/pipelines/publish-logs.yml
+  - job: Correctness_Bootstrap_Build_Framework
+    dependsOn: Determine_Changes
+    condition: or(ne(variables['Build.Reason'], 'PullRequest'), eq(dependencies.Determine_Changes.outputs['SetPathVars_compilers.containsChange'], 'true'))
```

Comment:
Going to change this condition to only run this bootstrap on CI. Double bootstrap on PR is pretty pointless. But this is an important test cause it's a customer scenario.